client/cmd/dexc-desktop/pkg/common.sh

```diff
@@ -0,0 +1,19 @@
+
+# This file defines common variables to be source'd by the various build scripts
+# in this directory.
+
+# pick up the release tag from git
+VER=$(git describe --tags --abbrev=0 --always | sed -e 's/^v//')
```

Comment:
Wow! I need to do some testing, but this is slick.

Reply:
The caveat with this one, though, is that it requires a tag on the repo clone being built (JoeGruff tripped over this above).  So for release builds it'll work but it won't work when only a commit hash is available due to the debian package builder requiring a version format x.y.z